src/Collapse.jsx

```diff
@@ -11,7 +12,8 @@ function toArray(activeKey) {
   return currentActiveKey;
 }
 
-const Collapse = React.createClass({
+const createReactClass = require('create-react-class');
```

Comment:
Very thanks for you contribution.

But, I think It is better to change the `createReactClass` code to class style component.